src/components/__tests__/FlawAffects.spec.ts

```diff
@@ -1,57 +1,49 @@
 import { describe, expect } from 'vitest';
-import { createTestingPinia } from '@pinia/testing';
-import { mount } from '@vue/test-utils';
 
 import FlawAffects from '@/components/FlawAffects.vue';
 
+import { mountWithConfig } from '@/__tests__/helpers';
+import type { ZodFlawType } from '@/types/zodFlaw';
+
 import { osimEmptyFlawTest, osimFullFlawTest } from './test-suite-helpers';
 
-createTestingPinia();
+const mountFlawAffects = (flaw: ZodFlawType) => mountWithConfig(FlawAffects, {
+  props: {
+    flawId: flaw.uuid,
+    embargoed: flaw.embargoed,
+    affects: flaw.affects,
+    affectsToDelete: [],
+    error: [],
+    affectCvssToDelete: {},
+  },
+});
 
 vi.mock('@/composables/useTrackers', () => ({
   useTrackers: vi.fn().mockReturnValue({
     trackersToFile: [],
+    isLoadingTrackers: true,
   }),
 }));
-describe('flawAffects', () => {
-  let subject;
 
+vi.mock('@/composables/useTrackers', () => ({
+  useTrackers: vi.fn().mockReturnValue({
+    trackersToFile: [],
+  }),
+}));
+describe('flawAffects', () => {
   osimEmptyFlawTest('Correctly renders the component when there are not affects to display', async ({ flaw }) => {
-    subject = mount(FlawAffects, {
-      props: {
-        flawId: flaw.uuid,
-        embargoed: flaw.embargoed,
-        affects: flaw.affects,
-        affectsToDelete: [],
-        error: [],
-      },
-    });
+    const subject = mountFlawAffects(flaw);
+
```

Comment:
As a general overview on tests (not specifically for this PR), I saw we use `subject` and `wrapper` for mounted components, I'd say we could choose one of them to use the same.

Reply:
Yes, normally I use wrapper because that is the name of the return type `VueWrapper`, the ones that were already using `subject` I did not rename them just to keep changes at a minimum